src/main/java/com/kodilla/ecommercee/controllers/CartController.java

```diff
@@ -0,0 +1,33 @@
+package com.kodilla.ecommercee.controllers;
+
+import com.kodilla.ecommercee.GenericEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@RequestMapping("/v1/cart")
+public class CartController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "getCart")
+    public GenericEntity getCartById(@RequestParam Long cartId) {
+        return new GenericEntity();
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "createCart", consumes = APPLICATION_JSON_VALUE)
+    public void createCart(@RequestBody GenericEntity cartDto) {
+    }
+
+    @RequestMapping(method = RequestMethod.PUT, value = "updateCart")
+    public GenericEntity updateCart(@RequestBody GenericEntity cartDto) {
+        return new GenericEntity();
+    }
+
+    @RequestMapping(method = RequestMethod.DELETE, value = "deleteCart")
+    public void deleteCartById(@RequestParam Long cartId) {
+
+    }
+
```

Comment:
Dziekuje ze zwrociles uwage na to ze endpointy byly zadane w zadaniu, nie dobierane dowolnie.
**1 utworzenie pustego koszyka** - createCart jest poprawne - OK
**2 pobranie elementów z pustego koszyka** - czyli pobranie koszyka po jego id - getCartById - OK
**3 dodanie elementów (produktów) do koszyka** - czyli miales poprawnie addProductsToCart (chodz to nietypowe, ze jest to wymagane w kontrolerze card, nie product). Czego brakuje w twojej starej implementacji, to parametr card. Dodajesz produkty do ktorego koszyka ?
**4 usunięcie konkretnego produktu z koszyka** - deleteCartById OK
**5 utworzenie zamówienia na podstawie koszyka** - createOrder - (to tez nietypowe ze to nie jest kontrolerze Orders..). Ten endpoint powinien jako parametr przyjmowac id card, nie orderDto.